src/bro/reasoner/openai_generic.py

```diff
@@ -55,6 +53,8 @@
 
 You are allowed and encouraged to create new user accounts on websites and services if it helps you complete the task.
 
+You are NOT ALLOWED to use `use_computer` tool until attempts using other tools fail.
+
```

Comment:
```suggestion
```

This seems to be a mistake; I removed this change from the previous changeset already.